gradle/versions.gradle

```diff
@@ -43,7 +43,7 @@ ext {
     powerMockVersion = "2.0.9"
     runnerVersion = "1.2.0"
     rulesVersion = "1.2.0"
-    robolectricVersion = "4.9.2"
+    robolectricVersion = "4.11.1"
```

Comment:
Do we need this?

Reply:
yes, our monthly pipeline UTs had OOM error again. So we decided to pick this up as well.